pxr/usd/usdGeom/mesh.h

```diff
@@ -54,28 +54,69 @@ class SdfAssetPath;
 
 /// \class UsdGeomMesh
 ///
-/// Encodes a mesh surface whose definition and feature-set
-/// will converge with that of OpenSubdiv, http://graphics.pixar.com/opensubdiv/docs/subdivision_surfaces.html.
-/// Certain interpolation ("tag") parameters are not yet supported.
+/// Encodes a mesh with optional subdivision properties and features.
 /// 
+/// Meshes are conceptually defined in terms of vertices, edges and faces.
+/// As a point-based primitive, the terms "vertex" and "point" are synonymous,
+/// as are associated terms such as "point index" and "vertex index", etc.
```

Comment:
I'd like to hear @mkuruc's take on this, but in all my documentation, I've been trying to draw a *distinction* between point and vertex, which is like an instancing relationship.  `points` defines the "pool of points", and a `vertex` associates one of those points with an edge/face... we wind up having many more vertices than we have points, typically, which follows (and I think, helps to make sense out of) from the fact that len(points) != sum_i(faceVertertexCounts) . And all of the attribute doc supports this interpretation.  So these statements contradict our other language.

There are other places below where you've equated points and vertices - I won't call them all out, but I'd request they all go.

Of course, if I'm the only one who thinks it's useful to draw this distinction, then let's talk!

Reply:
Maybe a clearer lexicon would be to just use `point` and `face vertex`, and not use simple `vertex` at all.  Using `vertex` when we have a `points` attribute is just confusing, witness all the places where you've parenthetically added it after you mention `points` ???

Reply:
I agree with Spiff about the importance of distinguishing between vertex and point.  (For example, summing up the faceVertexCounts vector will give you the total number of vertices, not the total number of points.

I've found the "point" vs. "face vertex" convention useful in the past.

Reply:
Okay, I will definitely get rid of the equivalence between point and vertex and use "point" exclusively.  I also prefer to use the term face-vertex.

I find it difficult (requiring discipline) to not use the term "vertex" when talking about meshes, but will do so here if needed.  The term "vertex" has unfortunately become ambiguous in the industry.  I'm coming form the OpenSubdiv perspective with these changes -- wanting to try and align the descriptions of the subdivision features -- and vertex is synonymous with point in OpenSubdiv.

Aside from the obvious multiple places to deal with point vs vertex usage, I'm now wondering if these leading statements in the introduction should be completely removed or replaced with something more accurate... I'll replace it with something that emphasizes avoidance of "vertex" in place of "point" and "face-vertex".

Reply:
I rephrased the opening paragraph in a recent commit to emphasize the use avoidance of the term "vertex" in favor "point" and "face-vertex", along with removing use of "vertex".  Let me know how that opening paragraph works for you now.

Reply:
This should be addressed in the recent commits -- please comment on them if necessary.